process-migration/image.yaml

```diff
@@ -0,0 +1,64 @@
+schema_version: 1
+
+name: "rhpam-7/rhpam75-process-migration-openshift"
+description: "Red Hat Process Automation Manager Process Migration 7.5 container image"
+version: "1.0"
+from: "ubi8:8-released"
+labels:
+    - name: "com.redhat.component"
+      value: "rhpam-7-rhpam75-process-migration-openshift-rhel8-container"
+    - name: "io.k8s.description"
+      value: "Platform for running Red Hat Process Automation Manager Process Migration"
+    - name: "io.k8s.display-name"
+      value: "Red Hat Process Automation Manager Process Migration 7.5"
+    - name: "io.openshift.expose-services"
+      value: "8080:http"
+    - name: "io.openshift.tags"
+      value: "javaee,rhpam,rhpam7"
```

Comment:
do we need rhpam7?

Reply:
following the same pattern as for other images